src/keys/rsa.js

```diff
@@ -127,9 +127,39 @@ function unmarshalRsaPublicKey (bytes) {
   return new RsaPublicKey(key)
 }
 
-function generateKeyPair (bits) {
-  const p = rsa.generateKeyPair({bits})
-  return new RsaPrivateKey(p.privateKey, p.publicKey)
+function fastKeys (bits, cb) {
+  if (typeof window === 'undefined') {
+    let ursa
+    try {
+      ursa = require('ursa')
+    } catch (err) {
```

Comment:
Why? This is simply the check if the optional dependency is available to be used